### ISSUE TITLE
SOGo not selectable on x86_64 architecture

### ISSUE THREAD

**Issue by andyfsimon** [iredmail/iRedMail#151]
REQUIRED BASIC INFO OF YOUR IREDMAIL SERVER:

- iRedMail version (check /etc/iredmail-release): 1.5.1
- Deployed with iRedMail Easy or the downloadable installer? downloadable installer
- Linux/BSD distribution name and version: Debian 11
- Store mail accounts in which backend (LDAP/MySQL/PGSQL): any 
- Web server (Apache or Nginx): any
- Manage mail accounts with iRedAdmin-Pro? not relevant
- [IMPORTANT] Related original log or error message is required if you're experiencing an issue.

When installing with the latest release, SOGo is not selectable on 64bit architecture. I traced the issue to the dialog/web_applications.sh file:

```
    # SOGo team doesn't offer binary packages for arm platform.
    if [[ X"${OS_ARCH}" != X'i386' ]] && [[ X"${OS_ARCH}" == X'x86_64' ]]; then
        export DIALOG_SELECTABLE_SOGO='NO'
    fi

```
This excludes x86_64 architectures instead of ARM. Correct line IMHO should be:

```
    # SOGo team doesn't offer binary packages for arm platform.
    if [[ X"${OS_ARCH}" != X'i386' ]] && [[ X"${OS_ARCH}" != X'x86_64' ]]; then
        export DIALOG_SELECTABLE_SOGO='NO'
    fi

```

I tested on my install and it works fine.

**Comment by iredmail** [comment]
Thank you for helping. Would you like to send me a PR (pull request)? :)

**Comment by iredmail** [comment]
Fixed moment ago, also re-tagged iRedMail-1.5.1 with this fix.
Thanks again for helping. :)

**Comment by andyfsimon** [comment]
Sorry, nighttime here in Europe haha :) 
My pleasure anyway, your work is fantastic.
